Patches/MeetingHudPatch.cs

```diff
@@ -678,12 +678,22 @@ public static void ReturnChangedPva(PlayerVoteArea pva)
     }
 
     public static void TryAddAfterMeetingDeathPlayers(PlayerState.DeathReason deathReason, params byte[] playerIds)
-    {
+    {    
         var AddedIdList = new List<byte>();
         foreach (var playerId in playerIds)
+        try { 
+        
+            byte walnut = Convert.ToByte(playerId);
+            if (Susceptible.CheckReason.Contains(walnut))
```

Comment:
instead of `if (Susceptible.CheckReason.Contains(walnut))`
do
```cs
var pc = utils.GetPlayerById(playerId);
if (pc==null) return;
if (pc.Is(CustomRoles.Susceptible))
```